contracts/payloads/IGP30/PayloadIGP30.sol

```diff
@@ -0,0 +1,757 @@
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        /// Action 1: Set wBTC token config and market rate curve on liquidity.
+        action1();
+
+        /// Action 2: Deploy wBTC/USDC and wBTC/USDT vaults.
+        action2();
+
+        /// Action 3: call cast() - transfer 2 wBTC to Fluid Reserve contract from treasury.
+        action3();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Set wBTC token config and market rate curve on liquidity.
+    function action1() internal {
+        {
+            AdminModuleStructs.RateDataV1Params[]
+                memory params_ = new AdminModuleStructs.RateDataV1Params[](1);
+
+            params_[0] = AdminModuleStructs.RateDataV1Params({
+                token: wBTC_ADDRESS, // wBTC
+                kink: 80 * 1e2, // 80%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink: 5 * 1e2, // 5%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+
+            LIQUIDITY.updateRateDataV1s(params_);
+        }
+
+        {
+            AdminModuleStructs.TokenConfig[]
+                memory params_ = new AdminModuleStructs.TokenConfig[](1);
+
+            params_[0] = AdminModuleStructs.TokenConfig({
+                token: wBTC_ADDRESS, // wBTC
+                threshold: 0.3 * 1e2, // 0.3
+                fee: 10 * 1e2, // 10%
+                maxUtilization: 100 * 1e2
+            });
+
+            LIQUIDITY.updateTokenConfigs(params_);
+        }
+
+    }
+
+    /// @notice Action 2: Deploy wBTC/USDC and wBTC/USDT vaults.
+    function action2() internal {
+        VaultConfig memory vaultConfig = VaultConfig({
+            // user supply config for the vault on Liquidity Layer.
+            supplyToken: wBTC_ADDRESS,
+            supplyMode: 1, // Mode 1
+            supplyExpandPercent: 25 * 1e2, // 25%
+            supplyExpandDuration: 12 hours, // 12 hours
+            supplyBaseLimitInUSD: 5_000_000, // $5M
```

Comment:
What does it mean by InUSD. Where are we converting USD in token amount? I can't find anything on it.